lib/pluginmanager/install.rb

```diff
@@ -202,11 +202,16 @@ def install_gems_list!(install_list)
     bundler_options[:without] = [] if development?
     bundler_options[:rubygems_source] = gemfile.gemset.sources
     bundler_options[:local] = true if local?
-
-    output = LogStash::Bundler.invoke!(bundler_options)
-
+    output = nil
+    # Unfreeze the bundle when installing gems
+    Bundler.settings.temporary({:frozen => false}) do
+      output = LogStash::Bundler.invoke!(bundler_options)
+      output << LogStash::Bundler.genericize_platform.to_s
+    end
     puts("Installation successful")
   rescue => exception
+    puts exception
+    puts exception.backtrace
```

Comment:
Nit: Either puts to stderr:

```suggestion
    $stderr.puts exception
    $stderr.puts exception.backtrace
```

or remove the `puts` debugging:

```suggestion
```